src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/CourtType.java

```diff
@@ -2,7 +2,7 @@
 
 public enum CourtType {
     FIRST_TIER,
-    FTPA,
-    UTPA;
-
+    UPPER_TRIBUNAL,
```

Comment:
Do these enum values need to be tested ?

Reply:
I don't think it is needed. They just represents possible values for any future implementations.